devel/qbe/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                qbe
+version             1.0
+categories          devel
+platforms           darwin
```

Comment:
`platforms darwin` is implicit and no longer needs to be included in Portfiles. You can delete this line.